websites/L/Lazada/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "392970361047613441",
+		"name": "niqowi"
+	},
+	"service": "Lazada",
+	"description": {
+		"en": "Visit Lazada.com to get incredible bargains and stacking coupons for even more savings."
+	},
+	"url": "lazada.com",
+	"version": "1.0.0",
+	"logo": "https://lzd-img-global.slatic.net/g/tps/tfs/TB1PApewFT7gK0jSZFpXXaTkpXa-200-200.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px.